### ISSUE TITLE
Blank screen in Firefox Browser for Android

### ISSUE THREAD

**Issue by anantajitjg** [pipwerks/PDFObject#214]
**Firefox Browser for Android** is showing blank screen instead of showing the fallback content when PDF files are embedded using PDFObject. The same issue is replicated with [PDFObject examples](https://pdfobject.com/#examples) also.
Is this the expected result with **Firefox Browser for Android**?

**Browser Version:** 68.8.1
**Android Version:** 10
Replicated the same issue in multiple Android devices.

I have temporarily fixed this issue by modifying the pdfobject.js file and adding this check in `isFirefoxWithPDFJS` function.
```
if(!isFirefox){
	return false;
}
if(ua.indexOf("Mobile") !== -1 || ua.indexOf("Tablet") !== -1) {
    return false;
}
```
Is this the right approach?

**Comment by maqsudi** [comment]
Yeah ,this fragment of program is right ,but i must see fully codes

28 may 2020, C.A. 16:41 tarixində Anantajit JG <notifications@github.com>
yazdı:

**Comment by pipwerks** [comment]
Sorry for the delayed response.  

I don't own an Android device, so I can't say for certain what the Firefox browser on Android supports. But in iOS, with both iPhones and iPads, embedded PDFs are not properly supported on web pages. PDFObject will render the fallback link instead.

Leaving this issue open in case anyone with Android cares to join in and help test/troubleshoot

**Comment by pipwerks** [comment]
FYI I have opened a BrowserStack account to test across platforms and will post when I have more info. Thanks

**Comment by pipwerks** [comment]
This is fixed in 2.2.0. See test cases at https://pdfobject.com/dev/

I have confirmed it works across every Android device I tested via BrowserStack (Samsung, Xiomi, Google, etc)

2.2.0 will be released as soon as I can finish testing. Closing ticket.

**Comment by anantajitjg** [comment]
I have tested this on all my Android devices, it is working great 🎉 ! @pipwerks Thanks a lot for the fix 🙂
